### DISCUSSION TITLE
BalenaOS Install

### DISCUSSION THREAD

**Discussion by RafSchandevyl** [jlesage/docker-putty#9] [General]
Hi, 

I'm trying to get this running on balenaOS but the containers fails to start.  
[Logs]    [2023-08-05T07:43:30.244Z] [putty] [cont-init   ] 10-clean-tmp-dir.sh: executing...
[Logs]    [2023-08-05T07:43:30.253Z] [putty] [cont-init   ] 10-clean-tmp-dir.sh: rm: can't remove '/tmp/balena': Resource busy
[Logs]    [2023-08-05T07:43:30.254Z] [putty] [cont-init   ] 10-clean-tmp-dir.sh: rm: can't remove '/tmp/resin': Resource busy
[Logs]    [2023-08-05T07:43:30.259Z] [putty] [cont-init   ] 10-clean-tmp-dir.sh: terminated with error 1.

How can i make it run ?
Regards

**Comment by jlesage** [comment]
Can you share more details on how you created the container (your `docker run` command) ?
